api/userservice.yaml

```diff
@@ -1799,6 +1799,9 @@ components:
         mainTopicId:
           type: integer
           example: 15
+        gender:
```

Comment:
I would assume "age" would also be there...

Reply:
it is in the MandatorySessionDataDTO which is kind of "base" class for this